eZ/Publish/Core/MVC/Symfony/View/Builder/ContentViewBuilder.php

```diff
@@ -155,7 +155,7 @@ public function buildView(array $parameters)
 
         // deprecated controller actions are replaced with their new equivalent, viewAction and embedAction
         if (!$view->getControllerReference() instanceof ControllerReference) {
-            if (\in_array($parameters['_controller'], ['ez_content:viewLocation', 'ez_content:viewContent'])) {
+            if (\in_array($parameters['_controller'], ['ez_content:viewAction', 'ez_content:viewContent'])) {
                 $view->setControllerReference(new ControllerReference('ez_content:viewAction'));
             } elseif (\in_array($parameters['_controller'], ['ez_content:embedLocation', 'ez_content:embedContent'])) {
                 $view->setControllerReference(new ControllerReference('ez_content:embedAction'));
```

Comment:
@Nattfarinn please correct me if I'm wrong, but it feels to me that this entire `if-if-if-elseif` block was placed here entirely for BC and now can be removed. We didn't set here controller reference for `'ez_content:viewAction'` before, so I expect it always is already set at this point. 

Or in other words: now that deprecated controllers are gone, those conditions would never be met, so they're obsolete.